lib/tasks/import.rake

```diff
@@ -36,8 +36,39 @@ namespace :import do
     end
   end
 
+  desc 'import scc-docs'
```

Comment:
How about using Oktokit (https://github.com/octokit/octokit.rb?tab=readme-ov-file#additional-query-parameters) to fetch .md files from private repos with an access token?

Reply:
yes, good idea. will look into it today!